tests/test_host_check.py

```diff
@@ -3267,6 +3167,167 @@ def test_warn_igb_uio_enabled(self, capsys):
         assert result is CheckState.NEUTRAL
 
 
+class TestPCIDevices(_CheckTestBase):
+    """Tests for the PCI devices check."""
+
+    check_group = "xrd-vrouter"
+    check_name = "PCI devices"
+    deps = ["IOMMU"]
+    cmds = ["lshw -businfo -c network"]
+    files = []
+
+    def test_success_vfio_iommu(self, capsys):
```

Comment:
I don't see how these tests stub/mock which drivers are available? (vfio/uio)

Reply:
In fact, `check_pci_devices` no longer cares what drivers are available, so not sure why there are test cases distinguishing between the drivers?

Reply:
Test cases cover both the case that IOMMU is required (i.e. uio not supported and no-iommu not set) and not